Userland/Applications/PartitionEditor/main.cpp

```diff
@@ -64,7 +64,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto device_paths = get_device_paths();
 
     auto partition_model = PartitionEditor::PartitionModel::create();
-    TRY(partition_model->set_device_path(device_paths.first()));
+    auto found_it = device_paths.find_if([&](auto const& path) {
+        auto result = partition_model->set_device_path(path);
+        return !result.is_error();
+    });
+    // TODO give the user some way to create a partition table?
```

Comment:
We probably prefer `// FIXME:` over `// TODO`s.

Reply:
Cheers, good to be consistent :)